packages/css-data/src/__generated__/property-value-descriptions.ts

```diff
@@ -140,6 +146,7 @@ export const properties = {
   flexGrow:
     "Specifies the ability of a flex item to grow to fill available space",
   flexShrink: "Specifies the ability of a flex item to shrink if necessary",
+  sizing: "Specifies the ability of a flex item to grow or shrink",
```

Comment:
maybe we should call it flexChildSizing or something to be clear in this global context

Reply:
alternative is to keep this in a separate place because these are non-standard? by adding them here we are making no distinction if the properties here are correct css properties or there is some mistake, so we can't validate them

could be a separate file that exports just the cust custom things, then merges them upon export into one for the consumer